src/prosemirror-plugin/comms/send-to-backend.ts

```diff
@@ -114,6 +114,8 @@ export function getSendToBackend(editorId: string) {
         serializableState: serialize({
           doc: state.doc,
           selection: state.selection,
+          // @ts-ignore
+          config: state.config,
```

Comment:
note @gzou466 , i added this back, as the ui currently needs it, from memory this did not cause perf issues when we were modifying directly in node_modules